nft-market/pages/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { useRouter } from "next/router";
+import PageWrapper from "../components/pageWrapper/PageWrapper";
+import { useEffect, useState } from "react";
+import translations from "../constants/en.global.json";
+import { CalimeroSdk, WalletConnection } from "calimero-sdk";
+import { config } from "../utils/calimeroSdk";
+
+let walletConnectionObject: WalletConnection | undefined = undefined;
+
+export default function Home() {
+  const router = useRouter();
+  const [isSignedIn, setIsSignedIn] = useState(false);
+  const [accountId, setAccountId] = useState<string | null>("");
+
+  const login = async () => {
+    try {
+      await walletConnectionObject?.requestSignIn({
+        contractId: config.nftContract,
+      });
+      router.push("/buynfts");
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  const logout = () => {
+    walletConnectionObject?.signOut();
+    setIsSignedIn(false);
+  };
+
+  useEffect(() => {
+    if (isSignedIn && localStorage.getItem("accountId")) {
+      setAccountId(localStorage.getItem("accountId"));
+    }
+  }, []);
+
+  useEffect(() => {
+    const init = async () => {
+      const calimero = await CalimeroSdk.init(config).connect();
+      walletConnectionObject = new WalletConnection(
+        calimero,
+        config.nftContract
+      );
+      const signedIn = await walletConnectionObject?.isSignedInAsync();
+      const account = walletConnectionObject?.account();
+      if (account && signedIn) {
+        localStorage.setItem("accountId", account.accountId);
+      }
+      setIsSignedIn(signedIn);
+    };
+    init();
+  }, []);
+
+  useEffect(() => {
+    const absolute = window.location.href.split("?");
+    const url = absolute[0];
+    router.replace(url);
+  }, []);
+
+  return (
+    <PageWrapper
+      isSignedIn={isSignedIn}
+      title={translations.home.title}
+      nearLogin={login}
+      nearLogout={() => {
+        logout();
+        router.reload();
+      }}
+      nearSignedIn={isSignedIn}
+    ></PageWrapper>
```

Comment:
What will index page display?

Reply:
Added a some home page design. Check it out and give me feedback if it is good for now.